tests/api.spec.js

```diff
@@ -108,10 +117,10 @@ describe('API', () => {
       });
     });
   describe('products', () => {
-    const productToCreate = {name: "Test_Wine_Or_Cheese", description: "Test_Description", price: "$Test", imgURL: "imageUrl", inStock: true, category: "Test category"};
+    const productToCreate = {id: 0, name: "Test_Wine_Or_Cheese", description: "Test_Description", price: "2", imgURL: "imageUrl", inStock: true, category: "Test category"};
     describe('GET /', () => {
-      it('Just returns a list of all products in the database', async () => {
-        const product = { name: 'Grenache', inStock: true };
+      it('Returns a list of all products in the database', async () => {
+        const product = { id: 1, name: 'Grenache', inStock: true };
```

Comment:
Because id's are serialized, the defined id may not be needed.

Reply:
everywhere